packages/notifications/stories/5-Toasts.stories.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
+import { Story, Meta } from '@storybook/react';
+import { Button } from '@zendeskgarden/react-buttons';
+import {
+  ToastProvider,
+  IToastProviderProps,
+  useToast,
+  Notification,
+  Title,
+  Paragraph,
+  Close
+} from '@zendeskgarden/react-notifications';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import { Dropdown, Field, Label, Select, Menu, Item } from '@zendeskgarden/react-dropdowns';
+import { ToastPlacement } from '../src/elements/toaster/reducer';
+
+export default {
+  title: 'Components/Notifications/Toasts',
+  component: ToastProvider
+} as Meta;
+
+type VALIDATION_TYPE = 'success' | 'warning' | 'error' | 'info';
+
+const StyledRow = styled(Row)`
+  margin-bottom: ${p => p.theme.space.sm};
+`;
+
+const NotificationExample: React.FC = () => {
+  const [placement, setPlacement] = useState<ToastPlacement>('top-end');
+  const { addToast, removeAllToasts } = useToast();
+
+  const addNotification = useCallback(
+    (type: VALIDATION_TYPE = 'info') => {
+      addToast(
+        ({ close }) => (
+          <Notification style={{ width: 275 }} type={type}>
+            <Title>{type.charAt(0).toUpperCase() + type.slice(1)} notification</Title>
+            <Paragraph>
+              Turnip greens yarrow ricebean cauliflower sea lettuce kohlrabi amaranth water
+            </Paragraph>
+            <Close aria-label="Close Alert" onClick={() => close()} />
+          </Notification>
+        ),
+        {
+          autoDismiss: type === 'error' ? false : 5000,
```

Comment:
Can this `5000` value be expressed as a story control?

Reply:
I've added this as a knob to the example. I'm still disabling the `autoDismiss` for the error examples though, if that's ok?

Reply:
This makes sense to me.